.github/docker/Dockerfile.centreon-collect-alma9

```diff
@@ -48,6 +48,8 @@ dnf install -y cmake \
     nfpm \
     sudo
 
+dnf upgrade python3
+
```

Comment:
python3 should be already on latest version because it is installed just before this step